docs/changelog.rst

```diff
@@ -10,6 +10,7 @@ New features:
 
 * :doc:`/plugins/kodiupdate`: Now supports multiple kodi instances
   :bug:`4101`
+* Allow to configure which fields are used to find duplicates
```

Comment:
It's nice to link to the docs for the new option.

Reply:
Ok I thought this was for plugins only